esmvaltool/cmorizers/data/formatters/datasets/esacci_landcover.py

```diff
@@ -0,0 +1,162 @@
+                                       (iris.coords.DimCoord(
+                                           target_lons,
+                                           standard_name='longitude',
+                                           units='degrees'), 2)])
+
+    combined_cube.coord('latitude').guess_bounds()
+    combined_cube.coord('longitude').guess_bounds()
+
+    return combined_cube
+
+
+def regrid_fix(cube, glob_attrs, var_name, var_info):
+    """Regrid cube and fixes.
+
+    Regrids the cube, fixes metadata, coordinates and glob_attrs.
+
+    Parameters
+    ----------
+    cube: iris.cube.Cube
+          Data cube to be regridded.
+
+    vals: dict
+          Variable long_name.
+
+    glob_attrs: dict
+          Dictionary holding cube metadata attributes.
+
+    var_name: str
+          Variable name.
+
+    var_info: dict
+          Dictionary holding cube metadata attributes.
+
+    Returns
+    -------
+    cube: iris.cube.Cube
+        data cube regreidded and with fixed coordinates.
+    """
+    logger.info("Regridding cube for %s", var_name)
+    regridded_cube = regrid_iris(cube)
+    fix_var_metadata(regridded_cube, var_info)
+    regridded_cube = fix_coords(regridded_cube)
+    set_global_atts(regridded_cube, glob_attrs)
+
+    return regridded_cube
+
+
+def cmorization(in_dir, out_dir, cfg, cfg_user, start_date, end_date):
+    """Cmorize data."""
+    glob_attrs = cfg['attributes']
+    if not start_date:
+        start_date = datetime(1992, 1, 1)
+    if not end_date:
+        end_date = datetime(2020, 12, 31)
+
+    for year in range(start_date.year, end_date.year + 1):
+        inpfile_pattern = os.path.join(in_dir, cfg['filename'])
+        year_inpfile_pattern = inpfile_pattern.format(year=year)
+        inpfiles = sorted(glob.glob(year_inpfile_pattern))
+        for inpfile in inpfiles:
+            cubes = iris.load(inpfile)
+            for var_name, vals in cfg['variables'].items():
+                var_info = cfg['cmor_table'].get_variable(vals['mip'],
+                                                          var_name)
+                glob_attrs['mip'] = vals['mip']
+                if var_name == 'shrubFrac':
+                    cube = cubes.extract_cube('SHRUBS-BD') + \
+                        cubes.extract_cube('SHRUBS-BE') + \
+                        cubes.extract_cube('SHRUBS-ND') + \
+                        cubes.extract_cube('SHRUBS-NE')
+                elif var_name == 'treeFrac':
+                    cube = cubes.extract_cube('TREES-BD') + \
+                        cubes.extract_cube('TREES-BE') + \
+                        cubes.extract_cube('TREES-ND') + \
+                        cubes.extract_cube('TREES-NE')
+                else:
+                    cube = cubes.extract_cube(vals['long_name'])
+
+                    regridded_cube = regrid_fix(cube, glob_attrs,
+                                                var_name, var_info)
```

Comment:
```suggestion
                regridded_cube = regrid_fix(cube, glob_attrs,
                                            var_name, var_info)
```

Otherwise you will never get the `regridded_cube` for shrubFrac and treeFrac